### PR TITLE
Upstream animation-play-after-finish-restarts.html from Blink

### PR DESCRIPTION
**Pull request by alancutter**
Upstream animation-play-after-finish-restarts.html from Blink

Import https://chromium.googlesource.com/chromium/src/+/f9c8bd86683623176fb87b8cdc84bac594c46176/third_party/WebKit/LayoutTests/web-animations-api/animation-play-after-finish-restarts.html

### CODE REVIEW

File: web-animations/interfaces/Animation/play.html [hunk 1]
```diff
@@ -31,5 +31,33 @@
 }, 'play() throws when seeking an infinite-duration animation played in ' +
    'reverse');
 
+test(function(t) {
+  var div = createDiv(t);
+  var animation = div.animate(null, 100 * MS_PER_SEC);
+  animation.currentTime = 10 * MS_PER_SEC;
+  assert_times_equal(animation.currentTime, 1 * MS_PER_SEC);
```
Comment (suzyh) [inline web-animations/interfaces/Animation/play.html:38]: Typo: 10 \* MS_PER_SEC here and below?

Comment (alancutter) [inline web-animations/interfaces/Animation/play.html:38]: I meant to change it to 1 \* MS_PER_SEC to make it harder to confuse with 100 \* MS_PER_SEC.

Comment (alancutter) [inline web-animations/interfaces/Animation/play.html:38]: Fixed.

**Comment by suzyh** [comment]
lgtm

**Pull request merged**